resources/migrations/001_update_migrations.yaml

```diff
@@ -8413,9 +8413,10 @@ databaseChangeLog:
               - column:
                   name: name
                   remarks: channel name
-                  type: ${text.type}
+                  type: varchar(254)
```

Comment:
what's the reason? :)

Reply:
h2 does not support unique index on text column